### PR TITLE
Fix documentation link to MockingOpenWithURL

### PR DESCRIPTION
**Pull request by avanderberg**
- [x] This is a small change 
- [ ] This change has been discussed in issue #<?> and the solution has been agreed upon with maintainers.

---

**Description:**

https://github.com/wix/Detox/blob/master/docs/APIRef.MockingOpenFromURL.md was renamed to
https://github.com/wix/Detox/blob/master/docs/APIRef.MockingOpenWithURL.md and displays an error 404 now.

### CODE REVIEW

File: docs/APIRef.DeviceObjectAPI.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ await device.uninstallApp('other.bundle.id');
 ### `device.openURL({url, sourceApp[optional]})`
 Mock opening the app from URL. `sourceApp` is an optional **iOS-only** parameter to specify source application bundle id.
 
-Read more in [Mocking Open From URL](APIRef.MockingOpenFromURL.md) section.
+Read more in [Mocking Open With URL](APIRef.MockingOpenWithURL.md) section.
```
Comment (LeoNatan) [inline docs/APIRef.DeviceObjectAPI.md:264]: Can you pls change to "with" instead of "With"? Thanks!

**Review by LeoNatan — CHANGES_REQUESTED** [review]
Thank you for your contribution. A small change and I will merge.

File: docs/APIRef.DeviceObjectAPI.md (hunk 1 above)
Comment (avanderberg) [reply docs/APIRef.DeviceObjectAPI.md:264]:
```suggestion
Read more in [Mocking Open with URL](APIRef.MockingOpenWithURL.md) section.
```

**Comment by LeoNatan** [comment]
Thank you!

**Pull request merged**